olive/workflows/run/config.py

```diff
@@ -16,7 +16,7 @@
 from olive.engine.packaging.packaging_config import PackagingConfig
 from olive.evaluator.olive_evaluator import OliveEvaluatorConfig
 from olive.model import ModelConfig
-from olive.passes import FullPassConfig, Pass
+from olive.passes import FullPassConfig
```

Comment:
I don't think there is a need to make the import of `Pass` lazy? You already made `Pass` importable by making ort imports lazy. 

Even if `Pass` is not imported at top level, the validator `validate_pass_search` gets called anyways even in setup mode in `run.py:lines 210-213`

Reply:
Also any imports of/from `olive.passes` will automatically trigger the `__init__` imports in all sub modules anyways

Reply:
yes, you are right. will revert the change